packages/web-react/src/components/Modal/demo/ModalScrollingLongContent.tsx

```diff
@@ -173,6 +183,32 @@ const ModalScrollingLongContent = () => {
           </ModalFooter>
         </ModalDialog>
       </Modal>
+
+      <Button onClick={toggleModalWithDropdown}>Open Non-Scrolling Modal with Dropdown</Button>
+
+      <Modal id="example-dropdown" isOpen={isModalWithDropdownOpen} onClose={handleModalWithDropdownClose}>
+        <ModalDialog isScrollable={false}>
+          <ModalHeader>Modal Title</ModalHeader>
+          <ModalBody>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam at excepturi laudantium magnam mollitia
+              perferendis reprehenderit, voluptate. Cum delectus dicta ducimus eligendi excepturi natus perferendis
+              provident unde. Eveniet, iste, molestiae?
+            </p>
+            <Dropdown id="dropdown-in-modal" renderTrigger={dropdownTrigger}>
```

Comment:
Shouldn't we be using `<DropdownModern>`? This dropdown will be deleted in version 2.

Reply:
You are right, I just got lost in all those deprecations and didn't know which Dropdown to use… 🙈